pages/[templateId].tsx

```diff
@@ -62,6 +62,7 @@ const MarketplaceTemplateDetail = (): JSX.Element => {
   const [isLoading, setIsLoading] = useState<boolean>(true);
   const [error, setError] = useState<string>('');
   const [saleId, setSaleId] = useState('');
+  const [serialNumber, setSerialNumber] = useState('');
```

Comment:
Let's use the sale's `assetId` instead because `asset_id` is built-in query parameter for the `/sales` endpoint.